docker-alfresco/Dockerfile

```diff
@@ -42,6 +42,13 @@ RUN java -jar /usr/local/tomcat/alfresco-mmt/alfresco-mmt*.jar install \
 
 # Docker CMD from parent image starts the server
 
+# Unpack ROOT.war
+RUN mkdir /usr/local/tomcat/webapps/ROOT && cd /usr/local/tomcat/webapps/ROOT && jar -xvf /usr/local/tomcat/webapps/ROOT.war && rm -f /usr/local/tomcat/webapps/ROOT.war && \
```

Comment:
Is it possible to make the directory read-only afterwards?

Reply:
Done.